test/check/check.go

```diff
@@ -245,6 +277,21 @@ func CheckForLeftoverObjects(currentVersion string) {
 	err = framework.Global.Client.List(context.Background(), &serviceAccounts, &listOptions)
 	Expect(err).NotTo(HaveOccurred())
 	Expect(serviceAccounts.Items).To(BeEmpty(), "Found leftover objects from the previous operator version")
+
+	services := corev1.ServiceList{}
+	err = framework.Global.Client.List(context.Background(), &services, &listOptions)
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
Expect(framework.Global.Client.List(context.Background(), &services, &listOptions)).To(Succees())

Reply:
done